crates/client/src/client/schema/snapshots/fuel_core_client__client__schema__chain__tests__chain_gql_query_output.snap

```diff
@@ -82,7 +82,6 @@ query {
         version
         add
         addi
-        aloc
```

Comment:
could this break older SDK versions loading the gas costs for predicate estimation?